keyboards/fortitude60/keymaps/default/keymap.c

```diff
@@ -0,0 +1,165 @@
+ * |      |  F7  |  F8  |  F9  |  F10 |  F11 |      |      |  F12 |ISO # |ISO / |      |      |      |
+ * `-------------+------+------+------+------+------+------+------+------+------+------+-------------'
+ *               |      |      |      |      |      |      |      |      |      |      |
+ *               `---------------------------------------------------------------------'
+ */
+[_RAISE] = LAYOUT( \
+  KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                       KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_DEL,  \
+  KC_GRV,  KC_1,    KC_UP,   KC_3,    KC_4,    KC_5,                       KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_BSPC, \
+  KC_DEL,  KC_LEFT, KC_DOWN, KC_RGHT, KC_F4,   KC_F5,                      KC_F6,KC_MINS,  KC_EQL, KC_LBRC, KC_RBRC,    KC_BSLS, \
+  _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  _______, _______,  KC_F12,  KC_NUHS, KC_NUBS, _______, _______, _______, \
+                    _______, _______, _______, _______, _______, _______,  _______, _______, _______, _______\
+),
+
+/* Adjust (Lower + Raise)
+ * ,-----------------------------------------.             ,-----------------------------------------.
+ * |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |             |  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |
+ * |------+------+------+------+------+------.             ,------+------+------+------+------+------|
+ * |      | Reset|RGB Tg|RGB Md|Hue Up|Hue Dn|             |Sat Up|Sat Dn|Val Up|Val Dn|      | Bksp |
+ * |------+------+------+------+------+------.             ,------+------+------+------+------+------|
+ * |      |      |      |Aud on|Audoff|AGnorm|             |AGswap|      |      |      |      |      |
+ * |------+------+------+------+------+------+-------------+------+------+------+------+------+------|
+ * |      |      |      |      |      |      |      |      |      |      |      |      |      |      |
+ * `-------------+------+------+------+------+------+------+------+------+------+------+-------------'
+ *               |      |      |      |      |      |      |      |      |      |      |
+ *               `---------------------------------------------------------------------'
+ */
+[_ADJUST] =  LAYOUT( \
+  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,                     KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12, \
+  _______, RESET  , RGB_TOG, RGB_MOD, RGB_HUD, RGB_HUI,                   RGB_SAD, RGB_SAI, RGB_VAD, RGB_VAI, _______, KC_BSPC, \
+  _______, _______, _______, AU_ON,   AU_OFF,  AG_NORM,                   AG_SWAP, _______, BL_TOGG, BL_STEP, BL_BRTG, _______, \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______ \
+)
+
+
+};
+
+void persistent_default_layer_set(uint16_t default_layer) {
```

Comment:
This function isn't needed anyhow. 

It's been replaced by: `set_single_persistent_default_layer(x)`